### ISSUE TITLE
Update RTs from p8 to GFSv17

### ISSUE THREAD

**Issue by dpsarmie** [ufs-community/ufs-weather-model#2311]
## Description
Regression test cases were previously updated from v16 to v17_p8. The p8 version of GFS is being phased out and now those regression tests need to be updated for GFSv17. 
In addition, new initial conditions will also need to be created for these updated tests.
Once completed, the _export_fv3_v16_ section in **default_vars.sh** should be ready to be phased out.

## Solution
New baselines for the p8 regression tests will be created and named gfsv17. Initial conditions will also be generated for these new cases.

**Comment by gspetro-NOAA** [comment]
@dpsarmie Is Issue #1653 still needed? It seems like perhaps the work in that issue has been done or will be covered by this issue?

**Comment by dpsarmie** [comment]
> [@dpsarmie](https://github.com/dpsarmie) Is Issue [#1653](https://github.com/ufs-community/ufs-weather-model/issues/1653) still needed? It seems like perhaps the work in that issue has been done or will be covered by this issue?

Yes, go ahead and close #1653 and we will leave this one open for now to cover the RT updates. Thanks!
